### ISSUE TITLE
Expand Anand Karaj / Sikh Marriage deck

### ISSUE THREAD

**Issue by maneetpaul** [KhalisFoundation/sttm-desktop#233]
To include: 

- keeta loriye kam
- dhan phir eh na
- palle taidai laagee
- vyah hoya mere babula
- pooree aasa ji mansa mere raam

**Comment by ManjotS** [comment]
This was done at the hackathon and is the database, unfortunately the wrong thing was surfaced to the UI. We are just waiting on that.

What is in the DB is the deck used by a few gurdwaras.

**Comment by maneetpaul** [comment]
I am going to open a new ticket for this along with other ceremonies
